common/app/uiComponent/core/RenderingActor.scala

```diff
@@ -0,0 +1,23 @@
+package uiComponent.core
+
+import akka.actor.Actor
+import model.ApplicationContext
+import uiComponent.UIComponent
+import scala.util.Try
+
+case class Rendering(component: UIComponent, ac: ApplicationContext)
+
+case class RenderingException(error: String) extends RuntimeException(error)
+
+class RenderingActor extends Actor with JavascriptRendering {
+
+  override def javascriptFile: String = "ui.bundle.server.js"
```

Comment:
I'm not that familiar with the client-side part of this - what would `ui.bundle.server.js` be?